### DISCUSSION TITLE
Remove www from the user name

### DISCUSSION THREAD

**Discussion by dauand** [Automattic/wordpress-activitypub#385] [Q&A]
Can www subdomain be dropped from the user name?

**Comment by pfefferle** [comment]
this is part of the next release.

**Reply by nichu42** [reply]
When can we expect the new release? 🤗

**Reply by ollrich** [reply]
Is it already implemented, or what was meant with next release?

**Reply by stuehmer** [reply]
For those who came here, more discussion is to be found there: https://wordpress.org/support/topic/drop-www/ , basically saying that it doesn’t matter if the UI shows the `www.` in the identifier, you can simply use it without! WebFinger will recognize both names.
